### DISCUSSION TITLE
problem with import

### DISCUSSION THREAD

**Discussion by ingfzamora** [dbeaver/dbeaver#11433] [Q&A] [answered]
Hello, 

First of all I love your work guys its awesome but I have an issue when I tried to import  a CSV into a table dbeaver fails and it cannot finish the import the problem is in a field like this one

**"Robo de cables, tubos y otros objetos destinados a servicios públicos Con violencia"**

As you can see the text has a comma and dbeaver try to sepparate it in a new column, Can any one help me to know how to import this csv into my table?

An example

Headers --> **Año,Clave_Ent,Entidad,Cve. Municipio,Municipio,Bien jurídico afectado,Tipo de delito,Subtipo de delito,Modalidad,Enero,Febrero,Marzo,Abril,Mayo,Junio,Julio,Agosto,Septiembre,Octubre,Noviembre,Diciembre**
2015,1,Aguascalientes,1001,Aguascalientes,El patrimonio,Robo,Robo de maquinaria,**"Robo de cables, tubos y otros objetos destinados a servicios públicos Con violencia"**,0,0,0,0,0,0,0,0,0,0,0,0

**Comment by LonwoLonwo** [comment, marked as answer]
As a workaround you can use another symbol in csv as delimiter. like ";'.
And then, when importing, specify a different separator in the settings.

![2021-03-02 11_35_10-DBeaver Enterprise 21 0 0 - _localhost_ Script-6](https://user-images.githubusercontent.com/45152336/109621346-e8987000-7b4b-11eb-9079-45bed44a9eef.png)

**Reply by ingfzamora** [reply]
thanks I'm going to try that
